evm/tests/log_opcode.rs

```diff
@@ -444,7 +444,7 @@ fn test_log_with_aggreg() -> anyhow::Result<()> {
     // Preprocess all circuits.
     let all_circuits = AllRecursiveCircuits::<F, C, D>::new(
         &all_stark,
-        &[16..17, 13..16, 15..18, 14..15, 9..10, 12..13, 17..20],
+        &[16..17, 13..16, 15..18, 14..15, 10..11, 12..13, 17..20],
```

Comment:
What was this change about?

Reply:
KeccakSponge table uses 3 or 4 more rows here, but we were already on the edge of 512 (2^9), hence the bump